chia-exporter won't start with existing chia installations which do not have the seeder config section (regression?)

versions:
- exporter: 0.2.0
- chia: latest main
- docker container with python 3.9, Debian 11

```
ERRO[0000] Error creating http client: error loading crawler config: missing private key or cert. Ensure config.yaml is up to date with the latest changes
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x8fc788]

goroutine 48 [running]:
github.com/chia-network/chia-exporter/internal/metrics.(*HarvesterServiceMetrics).httpGetPlots(0xc0002c6de0)
        /build/internal/metrics/harvester.go:62 +0x68
github.com/chia-network/chia-exporter/internal/metrics.(*HarvesterServiceMetrics).InitialData(0xc000053f08)
        /build/internal/metrics/harvester.go:56 +0x19
github.com/chia-network/chia-exporter/internal/metrics.(*Metrics).OpenWebsocket(0xc0002a4660)
        /build/internal/metrics/metrics.go:194 +0x1c9
github.com/chia-network/chia-exporter/cmd.startWebsocket(0xc0002c7560)
        /build/cmd/serve.go:54 +0x7e
created by github.com/chia-network/chia-exporter/cmd.glob..func1
        /build/cmd/serve.go:29 +0x15b
```

It works with a fresh new chia installation. I think there could be a problem with the config.yaml migration logic within chia-blockchain (if such a logic exists). I also think we had this issue a few weeks/months ago.

--- a/go.mod
+++ b/go.mod
@@ -3,7 +3,7 @@ module github.com/chia-network/chia-exporter
 go 1.17
 
 require (
-	github.com/chia-network/go-chia-libs v0.0.7
+	github.com/chia-network/go-chia-libs v0.0.8
 	github.com/oschwald/maxminddb-golang v1.8.0
 	github.com/prometheus/client_golang v1.12.0
 	github.com/sirupsen/logrus v1.8.1

--- a/internal/metrics/fullnode.go
+++ b/internal/metrics/fullnode.go
@@ -337,10 +337,8 @@ func setGaugeToFilesize(file string, g *wrappedPrometheus.LazyGauge) error {
 		if errors.Is(err, os.ErrNotExist) {
 			log.Debugf("file: chia_full_node file doesn't exist: %s\n", file)
 			return nil
-		} else {
-			return err
 		}
-
+		return err
 	}
 
 	g.Set(float64(fi.Size()))

--- a/internal/metrics/fullnode.go
+++ b/internal/metrics/fullnode.go
@@ -2,6 +2,7 @@ package metrics
 
 import (
 	"encoding/json"
+	"errors"
 	"fmt"
 	"os"
 	"time"
@@ -330,10 +331,16 @@ func (s *FullNodeServiceMetrics) RefreshFileSizes() {
 }
 
 func setGaugeToFilesize(file string, g *wrappedPrometheus.LazyGauge) error {
-	log.Debugf("Getting filesize of %s\n", file)
+	log.Debugf("file: chia_full_node Getting filesize of %s\n", file)
 	fi, err := os.Stat(file)
 	if err != nil {
-		return err
+		if errors.Is(err, os.ErrNotExist) {
+			log.Debugf("file: chia_full_node file doesn't exist: %s\n", file)
+			return nil
+		} else {
+			return err
+		}
+
 	}
 
 	g.Set(float64(fi.Size()))

--- a/internal/metrics/harvester.go
+++ b/internal/metrics/harvester.go
@@ -10,7 +10,6 @@ import (
 	log "github.com/sirupsen/logrus"
 
 	wrappedPrometheus "github.com/chia-network/chia-exporter/internal/prometheus"
-	"github.com/chia-network/chia-exporter/internal/utils"
 )
 
 // Metrics that are based on Harvester RPC calls are in this file
@@ -60,7 +59,11 @@ func (s *HarvesterServiceMetrics) httpGetPlots() {
 	// get_plots seems to sometimes not respond on websockets, so doing http request for this
 	log.Debug("Calling get_plots with http client")
 	plots, _, err := s.metrics.httpClient.HarvesterService.GetPlots()
-	utils.LogErr(plots, nil, err)
+	if err != nil {
+		log.Warnf("Could not get plot information from harvester: %s\n", err.Error())
+		return
+	}
+
 	s.ProcessGetPlots(plots)
 }
 

